src/client/components/ReferenceAutocomplete/ReferenceAutocomplete.react.js

```diff
@@ -132,6 +133,7 @@ class ReferenceAutocomplete extends React.Component {
             maxWidth: `${maxWidth}px`,
           }
         },
+        ...styles
```

Comment:
Property 'multiValueLabel' would be fully replaced is it is defined by the user. I propose to use composition, so user still overrides styles provided by react-select and our Reference Autocomplete component. E.g. something like this:

```js
    var {
        multiValueLabel: customMultiValueLabel = (provided, state) => provided, 
        ...otherStyles
    } = styles;
    
    const defaultMultiValueLabel = (provided, state) => {
      const rMargin = 150;
      const defaultMinWidth = 200;
      const { width = (rMargin + defaultMinWidth) } = this.props;
      const maxWidth = Math.max(width - rMargin, defaultMinWidth);

      return {
        ...provided,
        maxWidth: `${maxWidth}px`,
      }
    };
    
    //...
    
    styles: {
        otherStyles,
        multiValueLabel: (provided, state) => {
          customMultiValueLabel(defaultMultiValueLabel(provided, state), state)  
        } 
    }
```